Sources/DeepLearning/Layer.swift

```diff
@@ -1338,10 +1338,13 @@ public struct SimpleRNNCell<Scalar: TensorFlowFloatingPoint>: RNNCell {
 /// An LSTM Cell.
 public struct LSTMCell<Scalar: TensorFlowFloatingPoint>: RNNCell {
     public var inputWeight: Tensor<Scalar>
```

Comment:
Would be a little cleaner if you define all weights one the same line, and all biases on the same line.
```swift
var inputWeight, updateWeight, ...: Tensor<Scalar>
var inputBias, updateBias, ...: Tensor<Scalar>
```

Reply:
Done.